core/src/main/scala/com/databricks/labs/remorph/parsers/tsql/TSqlRelationBuilder.scala

```diff
@@ -1,18 +1,74 @@
 package com.databricks.labs.remorph.parsers.tsql
 
-import com.databricks.labs.remorph.parsers.{IncompleteParser, intermediate => ir}
 import com.databricks.labs.remorph.parsers.tsql.TSqlParser._
+import com.databricks.labs.remorph.parsers.{IncompleteParser, intermediate => ir}
 
-import scala.collection.JavaConverters._
+import scala.collection.JavaConverters.asScalaBufferConverter
 
 class TSqlRelationBuilder extends TSqlParserBaseVisitor[ir.Relation] with IncompleteParser[ir.Relation] {
 
   protected override def wrapUnresolvedInput(unparsedInput: String): ir.Relation = ir.UnresolvedRelation(unparsedInput)
 
-  override def visitTableSourceItem(ctx: TableSourceItemContext): ir.Relation = {
-    val table = ctx.fullTableName().getText
-    // [TODO]: Handle Table Alias ctx.tableAlias().getText
-    ir.NamedTable(table, Map.empty, is_streaming = false)
+  override def visitSelectStatementStandalone(ctx: TSqlParser.SelectStatementStandaloneContext): ir.Relation = {
+
+    // TODO: Process ctx.WithExpression
+    ctx.selectStatement().accept(this)
+    // val tableSources = Option(rawExpression.tableSources()).fold[ir.Relation](ir.NoTable())
+    // (_.accept(new TSqlRelationBuilder))
+  }
+
+  override def visitSelectStatement(ctx: TSqlParser.SelectStatementContext): ir.Relation = {
+    // TODO: val orderByClause = Option(ctx.selectOrderByClause).map(_.accept(this))
+    // TODO: val forClause = Option(ctx.forClause).map(_.accept(this))
+    // TODO: val optionClause = Option(ctx.optionClause).map(_.accept(this))
+
+    ctx.queryExpression.accept(this)
+  }
+
+  override def visitQuerySpecification(ctx: TSqlParser.QuerySpecificationContext): ir.Relation = {
+
+    // TODO: Process all the other elements of a query specification
+
+    val columns =
+      ctx.selectListElem().asScala.toList.map(_.accept(new TSqlExpressionBuilder()))
```

Comment:
`toList` isn't necessary here (although its absence may result in the IDE mistakenly flagging the line as erroneous).

Reply:
IDE is fine without the toList